mkdocs.yml

```diff
@@ -109,6 +109,11 @@ nav:
     - user_guide/backup-restore.md
     - Cloud Sync: user_guide/cloud-storage-team-workspaces.md
     - user_guide/privacy-mode.md
+  - Plugin Development:
```

Comment:
You need a 'Plugins' page in the user guide for **users** of plugins, detailing what they are, who they're available to, and how to install them